types/src/pull-schemas.ts

```diff
@@ -0,0 +1,49 @@
+import * as fs from "fs";
+import download from "download";
+import AdmZip from "adm-zip";
+import { program } from "commander";
+
+
+async function main() {
+
+    program.name("Desmos contract schemas downloader")
+        .description("Utility script to download the desmos contract json schemas from the github repository")
+        .option("--branch <branch>", "branch from from which they will be downloaded")
+        .option("--tag <tag>", "git tag from from which they will be downloaded")
+        .option("--commit <commit>", "commit id from from which they will be downloaded")
+        .action(async ({branch, tag, commit}) => {
+            let url;
+            if (branch !== undefined) {
+                url = `https://github.com/desmos-labs/desmos-contracts/archive/refs/heads/${branch}.zip`
+            } else if (tag !== undefined) {
+                url = `https://github.com/desmos-labs/desmos-contracts/archive/refs/tags/${tag}.zip`
+            } else if (commit !== undefined) {
+                url = `https://github.com/desmos-labs/desmos-contracts/archive/${commit}.zip`
+            } else {
+                // Fallback to master
+                url = `https://github.com/desmos-labs/desmos-contracts/archive/refs/heads/master.zip`;
+            }
+
+            console.log("Downloading archive from", url);
+            const zip = new AdmZip(await download(url));
+
```

Comment:
I think we should remove then create new schemas dir first to avoid the issue that remaining the removed schemas.
Say, if POAP manager contract used to have `state.json` and `state.json` is remove in the new version, then `pull-schema` would not remove it and it might occur the issue in the future.